R/generate_from_dgp.R

```diff
@@ -27,19 +28,30 @@ generate_from_dgp <- function(dgp, n_datasets, ...) {
 
```

Comment:
Should we assert that the values returned by the dgp are of a certain format, or is it better to leave this more general?

Reply:
Discussed with Noa and agreed that keeping things general is better here.